tests/model_forms/tests.py

```diff
@@ -855,6 +855,25 @@ def test_unique_together(self):
         self.assertEqual(len(form.errors), 1)
         self.assertEqual(form.errors['__all__'], ['Price with this Price and Quantity already exists.'])
 
+    def test_unique_together_exclusion(self):
+        """
+        Forms should not validate unique_together constraints when only part of
+        the contraint is included in fields.
+
+        Ensures support of commit=False usage. #13091
+        """
+        class BookForm(forms.ModelForm):
+            class Meta:
+                model = DerivedBook
+                fields = ('isbn', 'suffix1',)
+
+        DerivedBook.objects.create(isbn='12345')
+        form = BookForm({'isbn': '56789', 'suffix1': '0'})
+        self.assertTrue(form.is_valid(), form.errors)
+        instance = form.save(commit=False)
```

Comment:
I'd use `obj =` although I'm not sure the rest of the test is necessary besides demonstrating the usage. It might be worth adding an assertion for `DerivedBook._meta.unique_together` to ensure that's not changed in a way that would invalidate the test. Can you use `Book` instead of `DerivedBook`? Might be a bit simpler to understand.

Reply:
> Can you use Book instead of DerivedBook? Might be a bit simpler to understand.

I needed a unique_together field with a default, otherwise it gets skipped, and the `DerivedBook` ones are the only candidates. (I originally adjusted `Price.quanity` but ended up breaking other tests...)